lib/BoundaryClickWatcher/README.md

```diff
@@ -0,0 +1,40 @@
+#BoundaryClickWatcher
+Wrap a component in BoundaryClickWatcher to detect clicks on our outside of the component. See an example of this in the Conversation story.
+
+##Usage
+
+BoundaryClickWatcher has a boolean (boundaryIsActive) which is set to true when it detects an inner click and back to false when it detects a click outside of the boundary. The value of this can be passed onto the child component like so:
```

Comment:
I'd say its worth mentioning that it uses the render prop pattern and keep this simple. Something like: "`BoundaryClickWatcher` uses the render callback pattern and returns a boolean."

I think the usage shows just how to use it 😃